commercetools-models/src/main/java/io/sphere/sdk/productdiscounts/ProductDiscountDraftBuilder.java

```diff
@@ -0,0 +1,71 @@
+package io.sphere.sdk.productdiscounts;
+
+import javax.annotation.Nullable;
+
+import io.sphere.sdk.models.Base;
+import io.sphere.sdk.models.Builder;
+import io.sphere.sdk.models.LocalizedString;
+
+public class ProductDiscountDraftBuilder extends Base implements Builder<ProductDiscountDraft> {
+
+    private LocalizedString name;
+    @Nullable
+    private LocalizedString description;
+    private ProductDiscountValue value;
+    private ProductDiscountPredicate predicate;
+    private String sortOrder;
+    private Boolean isActive;
```

Comment:
the field should be "active" https://en.wikipedia.org/wiki/JavaBeans#JavaBean_conventions